test/Datadog.Trace.ClrProfiler.IntegrationTests/SmokeTests/StackExchangeRedisStackOverflowExceptionSmokeTest.cs

```diff
@@ -23,5 +24,12 @@ public void NoExceptions()
 
             CheckForSmoke();
         }
+
+        protected override void AssertProcessResultIsSuccessful(ProcessResult result)
+        {
+            var successCodes = new HashSet<int> { 0, 139 };
```

Comment:
Oftentimes the StackExchange smoke test exits with a 139 exit code. If we see that exit code and nothing being written to standard error, I think we should consider the result passing. Thoughts?

Reply:
I don't think I agree. My first instinct was the same, but after talking with @lucaspimentel about it, it's something I think we need to look into.

Reply:
I would ultimately like to get to the bottom of this too, but it is a little frustrating that it seems to reproduce very frequently and our managed code is invoked just fine. I looked up the segfault issue and I found a possible related CoreCLR issue concerning a race condition that particularly affects Alpine (we're using the 2.1 SDK Alpine image). I don't really want to investigate this further at the moment, but perhaps we can test that hypothesis when the next 2.1 patch comes out.

Reply:
I also found Alpine mentioned when I was searching for 139 and corefx segfaults in docker. We can easily change the base docker image in `docker/dotnet.dockerfile`, if you want to try that. See https://hub.docker.com/_/microsoft-dotnet-core-sdk/

Reply:
https://github.com/dotnet/coreclr/issues/22176